tests/functests/instancetype_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/rand"
-	"k8s.io/utils/pointer"
+	"k8s.io/utils/ptr"
```

Comment:
why is this change necessary?

Reply:
Because `k8s.io/utils/pointer` is deprecated and `golangci-lint` complains about it.